sixtyfps_runtime/rendering_backends/default/Cargo.toml

```diff
@@ -14,7 +14,6 @@ path = "lib.rs"
 [features]
 x11 = ["sixtyfps-rendering-backend-gl/x11"]
 wayland = ["sixtyfps-rendering-backend-gl/wayland"]
-default = ["sixtyfps-rendering-backend-gl"]
```

Comment:
without that, i'm afraid that the gl backend might not be build by default for the rust crate

Reply:
for example the LSP might also need that, and the viewer

Reply:
Oh? Why not? It is for me, because api/sixtyfps-rs/Cargo.toml has this:

```toml
backend-gl = ["sixtyfps-rendering-backend-default/sixtyfps-rendering-backend-gl"]
backend-qt = ["sixtyfps-rendering-backend-default/sixtyfps-rendering-backend-qt"]
default = ["backend-gl", "backend-qt"]
```

Reply:
Yes, that is true, all "frontends". Another option is to keep it here but use `default-features = false` in the C++, Rust API and interpreter crate. Would that be okay?

Reply:
right,  and the interpreter do the same so it should be ok